lib/fastlane/plugin/yarn/helper/yarn_helper.rb

```diff
@@ -31,7 +31,8 @@ def check_install
 
         UI.message("Checking yarn install and dependencies")
         begin
-          Action.sh(self.yarn, print_command: true, print_command_output: true)
+          command = [self.yarn, "--version"].compact.join(" ")
```

Comment:
yes the latter, this is an example on how Im using it. I was trying to take the install process out of it and let the plugin handle installing dependencies automatically for you.
```ruby
  desc "Run React Native unit tests with code coverage"
  lane :test do
    yarn(
      task: "test:teamcity",
      package_path: "../package.json"
      )
  end

  desc "Bundle React Native library"
  lane :bundle do
    yarn(
      task: 'bundle-android',
      package_path: "../package.json"
    )
  end
```

But can see making a little more flexible and let people just run `yarn(task: 'install')`